server/pbench/bin/pbench-report-status

```diff
@@ -86,36 +75,52 @@ rm -rf $tmp" EXIT QUIT INT
 mkdir $tmp
 
 server=$(getconf.py --config $IDXCONFIG server Server)
-if [[ -z "$server" ]] ;then
-    echo "$prog: no server defined in [Server] section of $IDXCONFIG" >> ${tmp}/log
-    exit 2
-fi
+index_prefix=$(getconf.py --config $IDXCONFIG index_prefix Settings)
+if [ -z "$server" -o -z "$index_prefix" ] ;then
+    # We don't have an Elasticsearch configuration, use syslog.
+    if [ -z "$server" -o -z "$index_prefix" ] ;then
```

Comment:
Seems like this should be `if [ -z "$server" ]`

Reply:
Thanks, nice catch.  I'll fix this.